doc/users/install.rst

```diff
@@ -108,7 +108,9 @@ Interface Dependencies
 Nipype provides wrappers around many neuroimaging tools and contains some
 algorithms. These tools will need to be installed for Nipype to run. You can
 create containers with different versions of these tools installed using
-`Neurodocker <https://github.com/kaczmarj/neurodocker>`_
+Neurodocker_. Read the brief :doc:`neurodocker`.
```

Comment:
I would prefer that this be parenthetical, rather than a sentence. See above suggestion.